Content.Client/GameObjects/Components/Atmos/PipeVisualizer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Content.Shared.GameObjects.Components.Atmos;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+using Robust.Client.Interfaces.GameObjects.Components;
+using Robust.Client.Interfaces.ResourceManagement;
+using Robust.Client.ResourceManagement;
+using Robust.Shared.GameObjects.Components.Renderable;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+using Robust.Shared.Utility;
+using System;
+using System.Collections.Generic;
+using YamlDotNet.RepresentationModel;
+
+namespace Content.Client.GameObjects.Components.Atmos
+{
+    [UsedImplicitly]
+    public class PipeVisualizer : AppearanceVisualizer
+    {
+        private readonly List<object> _pipeLayerKeys = new List<object>();
```

Comment:
This seems unused, and making a list of `object` doesn't feel like the best idea.